deployment/gke/config_project.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
If you're not around, I cannot imagine navigating this to run the right thing :)

Should we have a deployment/gke/README.md to give an overview of what all these scripts are and the flow in general?  (follow-up PR after you figure all this out is fine)

Reply:
Yeah, sure. Also I can host a tech-talk for this, and do a live deployment?